tests/test_core.py

```diff
@@ -32,88 +32,89 @@ def accounts_response(fake: Faker):
             "code": 200,
             "is_success": True,
         },
-        "data": {
-            "id": 1000,
-            "name": fake.company(),
-        },
+        "data": [
+            {
+                "id": 1000,
+                "name": fake.company(),
+            },
+        ],
+        "extra": {},
```

Comment:
@mjsqu The [API docs](https://docs.getdbt.com/dbt-cloud/api-v2) seem to be outdated 😕, so can you check that these match to actual responses from the API? 🙏

Reply:
Hold on - I'll do a request to the accounts stream instead

Reply:
```json
{
  "status": {
    "code": 200,
    "is_success": true,
    "user_message": "Success!",
    "developer_message": ""
  },
  "data": [
    {
      "lock_reason": null,
      "unlock_if_subscription_renewed": false,
      "read_only_seats": 50,
      "docs_job_id": null,
      "freshness_job_id": null,
      "id": 1,
      "name": "Bananas",
      "state": 1,
      "plan": "enterprise",
      "pending_cancel": false,
      "run_slots": 15,
      "developer_seats": 50,
      "queue_limit": 50,
      "pod_memory_request_mebibytes": 1800,
      "run_duration_limit_seconds": 86400,
      "enterprise_authentication_method": "azure_ad",
      "enterprise_login_slug": "sso-azure",
      "enterprise_unique_identifier": null,
      "salesforce_customer_id": null,
      "billing_email_address": null,
      "locked": false,
      "develop_file_system": true,
      "unlocked_at": null,
      "created_at": "2021-02-24T16:29:13.050446+00:00",
      "updated_at": "2023-03-24T19:45:05.512126+00:00",
      "starter_repo_url": null,
      "sso_reauth": false,
      "force_sso": true,
      "git_auth_level": "personal",
      "identifier": "",
      "enterprise_login_url": ""
    }
  ],
  "extra": {
    "filters": {
      "pk__in": [
        1
      ]
    },
    "order_by": null,
    "pagination": {
      "count": 1,
      "total_count": 1
    }
  }
}
```